client/components/tinymce/plugins/contact-form/style.scss

```diff
@@ -0,0 +1,73 @@
+.editor-contact-form-modal {
+	padding: 0;
+}
+
+.editor-contact-form-modal.dialog.card {
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	max-width: 100%;
+
+	@include breakpoint( ">660px" ) {
+		top: 5%;
+		bottom: 5%;
+		left: 50%;
+		margin-left: -275px;
+		width: 550px;
+	}
+}
+
+.editor-contact-form-modal .dialog__content {
+	position: static;
+	color: $gray-dark;
+}
+
+.editor-contact-form-modal .section-nav {
+	padding: 0 16px;
+
+	@include breakpoint( ">660px" ) {
+		padding: 0;
+	}
+}
+
+.editor-contact-form-modal .dialog__action-buttons {
+	position: absolute;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	margin: 0;
+}
+
+.editor-contact-form-modal__secondary-actions {
+	float: left;
+	display: flex;
+	justify-content: center;
+	height: 40px;
+}
+
+button.foldable-card__action.foldable-card__expand .gridicon.gridicons-pencil {
```

Comment:
Couldn't this style apply outside the context of the editor usage? Seems like it should be scoped to the contact form.